django/chat/models.py

```diff
@@ -101,7 +101,7 @@ def from_defaults(self, mode=None, chat=None):
             from chat.utils import copy_options
 
             new_options = self.create()
-            copy_options(chat.user.default_preset.options, new_options)
+            copy_options(chat.user.default_preset.options, new_options, chat.user)
```

Comment:
When a new chat is created, the user might have a default preset selected which has a QA library that isn't accessible to them. We do this permission check inside `copy_options`.